model_analyzer/triton/server/server_config.py

```diff
@@ -45,6 +45,7 @@ class TritonServerConfig:
         'grpc-port',
         'grpc-infer-allocation-pool-size',
         'grpc-use-ssl',
+        'grpc-use-ssl-mutual',
```

Comment:
Just curious.. where are these configurations being set? I see the verification happening in `test_triton_server.py`.

Reply:
In the `triton_server_flags`:

https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#triton-server-flags